test/native/setup.js

```diff
@@ -242,6 +242,11 @@ jest.mock(
 jest.mock( 'react-native/Libraries/ActionSheetIOS/ActionSheetIOS', () => ( {
 	showActionSheetWithOptions: jest.fn(),
 } ) );
+jest.mock( 'react-native/Libraries/Linking/Linking', () => {
+	return {
+		addEventListener: jest.fn( () => ( { remove: jest.fn() } ) ),
+	};
+} );
```

Comment:
Thinking "out loud:" is there a way to avoid mocking and merely update the returned mock value? Not necessarily confident this is necessary or better, but I wondered if it is helpful to avoid re-mocking a module already managed by RN core. WDYT? 

```js
Linking.addEventListener.mockReturnValue({ remove: jest.fun() });
```

Reply:
Yep, definitely, we could only mock the return value of `addEventListener`.

Reply:
I applied your suggestion in https://github.com/WordPress/gutenberg/pull/51303/commits/97265c70ebe022868835bf476928e069e131461d.